crates/rpc/rpc/src/eth/gas_oracle.rs

```diff
@@ -79,6 +83,21 @@ impl GasPriceOracleConfig {
     }
 }
 
+type BlockHash = FixedBytes<32>;
+
+/// Wrapper struct for LruMap
+#[derive(Deref, DerefMut)]
+pub struct EffectiveTipLruCache(LruMap<BlockHash, (BlockHash, Vec<U256>), ByLength>);
+
+impl Debug for EffectiveTipLruCache {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        f.debug_struct("EffectiveTipLruCache")
+            .field("cache_length", &self.len())
+            .field("cache_memory_usage", &self.memory_usage())
+            .finish()
+    }
+}
```

Comment:
if we add a new type `GasPriceOracleInner` then we don't need this wrapper and can just use the LruMap as a field directly

Reply:
LruMap doesn't implement Debug natively. Do have any suggestions besides adding a wrapper as syntactic sugar to work around having to implement Debug for LruMap<> when its an external crate.

Reply:
can impl debug manually for GasPriceOracleInner